build_tools/packaging/linux/build_package.py

```diff
@@ -69,7 +69,7 @@ def create_deb_package(pkg_name, config: PackageConfig):
 
     Returns: None
     """
-
+    print("create_deb_package {}".format(pkg_name))
```

Comment:
nit: can this use an fstring instead?

```suggestion
    print(f"create_deb_package: {pkg_name}")
```

https://peps.python.org/pep-0498/ (and other references)